applications/stepswitch/src/stepswitch_bridge.erl

```diff
@@ -253,6 +268,7 @@ build_bridge(#state{endpoints=Endpoints
                     ,{<<"From-URI">>, bridge_from_uri(CIDNum, JObj)}
                     ,{<<"Reseller-ID">>, wh_services:find_reseller_id(AccountId)}
                    ]),
+    BLegEvents = wh_json:get_value(<<"B-Leg-Events">>, JObj, []),
```

Comment:
There is a `wh_json:get_list_value/3` that will return the value if it is a list, otherwise returns the Default. Using it helps the reader know to expect a list of things here.